web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,27 +1,59 @@
 <script lang="ts">
+	import { onMount } from 'svelte';
+	import { lazyLoad } from 'unlazy';
 	import { imageLoad } from '$lib/utils/image-load';
 
 	export let url: string;
 	export let altText: string;
 	export let heightStyle: string | undefined = undefined;
 	export let widthStyle: string;
+	export let thumbhash: string | null;
 	export let curve = false;
 	export let shadow = false;
 	export let circle = false;
 	let loading = true;
+
+	let imageElement: HTMLImageElement;
+
+	if (thumbhash != null) {
+		onMount(() => {
+			lazyLoad(imageElement, {
+				hash: thumbhash || '',
+				hashType: 'thumbhash'
+			});
+		});
+	}
```

Comment:
I know the `|| ''` is unnecessary but the linting hates it when it's not there.

Reply:
Only because the value could have changed between the if statement and when it gets evaluated. That won't be the case if it is immediately after the if statement.